vspc/server.py

```diff
@@ -118,78 +127,93 @@ def handle_vm_vc_uuid(self, socket, data):
+        )
         secret = os.urandom(4)
-        LOG.debug(">> %s VMOTION-GOAHEAD %s %s", peer, data, secret)
+        LOG.debug(
+            "VM-VC-UUID: %s >> %s VMOTION-GOAHEAD %s %s",
+            vm_uuid, peer, data, secret
+        )
         writer.write(IAC + SB + VMWARE_EXT + VMOTION_GOAHEAD +
                      data + secret + IAC + SE)
         await writer.drain()
 
-    async def handle_vmotion_peer(self, writer, data):
+    async def handle_vmotion_peer(self, writer, data, vm_uuid):
         socket = writer.get_extra_info('socket')
         peer = socket.getpeername()
-        LOG.debug("<< %s VMOTION-PEER %s", peer, data)
-        LOG.debug("<< %s VMOTION-PEER-OK %s", peer, data)
+        LOG.debug("VM-VC-UUID: %s << %s VMOTION-PEER %s", vm_uuid, peer, data)
+        LOG.debug(
+            "VM-VC-UUID: %s << %s VMOTION-PEER-OK %s",
+            vm_uuid, peer, data
+        )
         writer.write(IAC + SB + VMWARE_EXT + VMOTION_PEER_OK + data + IAC + SE)
         await writer.drain()
 
-    def handle_vmotion_complete(self, socket, data):
+    def handle_vmotion_complete(self, socket, data, vm_uuid):
         peer = socket.getpeername()
-        LOG.debug("<< %s VMOTION-COMPLETE %s", peer, data)
+        LOG.debug(
+            "VM-VC-UUID: %s << %s VMOTION-COMPLETE %s",
+            vm_uuid, peer, data
+        )
 
-    async def handle_do(self, writer, opt):
+    async def handle_do(self, writer, opt, vm_uuid):
         socket = writer.get_extra_info('socket')
         peer = socket.getpeername()
-        LOG.debug("<< %s DO %s", peer, opt)
+        LOG.debug("VM-VC-UUID: %s << %s DO %s", vm_uuid, peer, opt)
         if opt in (BINARY, SGA):
-            LOG.debug(">> %s WILL", peer)
+            LOG.debug("VM-VC-UUID: %s >> %s WILL", vm_uuid, peer)
             writer.write(IAC + WILL + opt)
             await writer.drain()
         else:
-            LOG.debug(">> %s WONT", peer)
+            LOG.debug("VM-VC-UUID: %s >> %s WONT", vm_uuid, peer)
             writer.write(IAC + WONT + opt)
             await writer.drain()
 
-    async def handle_will(self, writer, opt):
+    async def handle_will(self, writer, opt, vm_uuid):
         socket = writer.get_extra_info('socket')
         peer = socket.getpeername()
-        LOG.debug("<< %s WILL %s", peer, opt)
+        LOG.debug("VM-VC-UUID: %s << %s WILL %s", vm_uuid, peer, opt)
         if opt in (BINARY, SGA, VMWARE_EXT):
-            LOG.debug(">> %s DO", peer)
+            LOG.debug("VM-VC-UUID: %s; >> %s DO", vm_uuid, peer)
             writer.write(IAC + DO + opt)
             await writer.drain()
         else:
-            LOG.debug(">> %s DONT", peer)
+            LOG.debug("VM-VC-UUID: %s >> %s DONT", vm_uuid, peer)
             writer.write(IAC + DONT + opt)
             await writer.drain()
 
     async def option_handler(self, cmd, opt, writer, data=None):
         socket = writer.get_extra_info('socket')
+        uuid = self.sock_to_uuid.get(socket)
```

Comment:
Can we maybe do something like this
```python
from uuid import UUID
uuid = self.sock_to_uuid.get(socket)
if uuid:
    uuid = str(UUID(uuid))
```
This would convert the UUID to the format used by OpenStack and would thus make searching for specific instances much easier.

We could also safe it in the openstack format and then only convert it with `uuid.replace('-', '')` when writing to disk. Whatever you think is more suitable/performant.